run_docker.sh

```diff
@@ -10,48 +14,74 @@ usage() {
 
 # Add validation to given arguments. If the parameters are not set or are empty, show usage message.
 # TODO
-
-username="$1"
-password="$2"
+username=oleksandr6676
+password=q532484966q
```

Comment:
Also, can I somehow avoid printing my own password here? Docker suggests to use --password-stdin command, but it seems like I need to create one more file to store my password there